web/yo/app/scripts/app.spec.js

```diff
@@ -0,0 +1,428 @@
+                    },
+                    "description_uuid": "2be3d33e-daae-4b66-99bd-0c747cb77733",
+                    "level": "",
+                    "level_uuid": "b7ff9248-3bd1-4c1a-b55f-29e4cbd3f54c",
+                    "short": ""
+                },
+                "prognostic_uuid": "1924db33-3e49-4df9-a5ca-a5bdd9088f5e",
+                "summary": "There is promising clinical data in patients with AKT1 E17K mutant gynecological cancer treated with the pan-AKT targeted inhibitor AZD5363.",
+                "summary_review": {
+                    "updateTime": 1508177294799,
+                    "updatedBy": "Debyani Chakravarty"
+                },
+                "summary_uuid": "84b36b75-e817-47f0-bce3-c4d7f108a49b"
+            }]
+        }],
+        "name": "AKT1",
+        "name_comments": [{
+            "content": "Hi Andy, In your cleanup please make sure that PMIDs: 9843996, 23134728 are curated. Thanks so much!",
+            "date": "1425671517595",
+            "email": "debyani.c@gmail.com",
+            "resolved": "true",
+            "userName": "Debyani Chakravarty"
+        }],
+        "summary": "AKT1, an intracellular kinase, is mutated at low frequencies in a diverse range of cancers.",
+        "summary_comments": [{
+            "content": "AKT1 encodes an intracellular kinase which is a critical downstream effector in the oncogenic PI3K signaling pathway. Mutation of the AKT1 gene are observed at low frequencies (<5%) in a range of tumors including breast, lung and colorectal.",
+            "date": "1460659497528",
+            "email": "s.m.phillips2@gmail.com",
+            "resolved": "true",
+            "userName": "Sarah Phillips"
+        }, {
+            "content": "AKT1 encodes a serine/threonine kinase involved in metabolism, proliferation, cell survival, growth and angiogenesis. Mutations of AKT1 gene are found in breast, lung and colorectal cancers, among others.",
+            "date": "1460659543592",
+            "email": "s.m.phillips2@gmail.com",
+            "resolved": "true",
+            "userName": "Sarah Phillips"
+        }],
+        "summary_review": {
+            "updateTime": 1493325255069,
+            "updatedBy": 'Jiaojiao wang'
+        },
+        "summary_uuid": "7c2782d9-fbde-417b-817f-b16448b931f5",
+        "type": {
+            "ocg": "Oncogene",
+            "ocg_review": {
+                "updateTime": 1493325255069
+            },
+            "ocg_uuid": "55c0ed9b-477e-469b-ae8a-9a14572ae8ac",
+            "tsg": "",
+            "tsg_review": {
+                "updateTime": 1493325255069
+            },
+            "tsg_uuid": "ad6e2b65-f03c-4385-b5fa-e162ce26f2f6"
+        },
+        "type_uuid": "a3c1500e-46fb-4834-ae06-eadc6f6ca341"
+    }
+};
+OncoKB.config = {
+    "apiLink": "",
+    "curationLink": "",
+    "privateApiLink": "",
+    "publicApiLink": "",
+    "testing": true,
+    "production": false,
+    "firebaseConfig": {
```

Comment:
Currently, copy and paste config.json to here currently. We'll import config.json in here directly in the future.